harmony/harmonydb/sql/20240814-sectors-task-events.sql

```diff
@@ -0,0 +1,51 @@
+ALTER TABLE harmony_task_history
+    ADD COLUMN sp_id BIGINT DEFAULT NULL,
+    ADD COLUMN sector_number BIGINT DEFAULT NULL;
+
+CREATE INDEX harmony_task_history_sp_sector_index ON harmony_task_history (sp_id, sector_number);
+
+CREATE TABLE sectors_task_events (
+    sp_id BIGINT NOT NULL,
+    sector_number BIGINT NOT NULL,
+    events BYTEA ARRAY DEFAULT '{}',
```

Comment:
Or would just store the events as separate columns, or possibly even just keep the TaskID which later can be referenced from `harmony_task_history`

Reply:
Thank you for the response. This PR is not ready yet. I have some questions regarding the history table. If the history table record is designed to be deletable, I think maintaining a long-term relational table is necessary. If the history table record is not deleted, it will be much simpler.

Reply:
Yeah, I believe we want to keep the history longer-term, that's why it's there; We might add options for cleaning it up automatically, but the cleanup logic will be up to the SP to configure with the default being to keep this metadata.